sample-barebones-kotlin/src/main/java/com/facebook/samples/lithoktbarebones/BoxItemChangedEvent.kt

```diff
@@ -0,0 +1,10 @@
+package com.facebook.samples.lithoktbarebones
+
+import com.facebook.litho.annotations.Event
+
+@Event
+data class BoxItemChangedEvent(
```

Comment:
Thank you for providing a test case! 
If you feel empowered, you can add test to `litho-processor` module for an `EventGenerator`s methods. But no pressure - we plan to give our tests more love next year anyway, cleaning up the approach :)